src/components/Header/HeaderIdentity.tsx

```diff
@@ -61,7 +61,7 @@ const drawerWithChildren = (classes, label, logout, children) =>
         {logout}
     </HeaderCollapse>
 
-const HeaderIdentity: React.SFC<HeaderIdentityProps & React.HTMLAttributes<HTMLDivElement>> = ({ avatar, children, className, onLogout, username }) => {
+const HeaderIdentity: React.SFC<HeaderIdentityProps & React.HTMLAttributes<HTMLDivElement>> = ({ avatar, children, className, logoutLinkText = 'Log out', onLogout, username }) => {
```

Comment:
Shouldn't `logoutLinkText` be defined in `HeaderIdentityProps`?

Reply:
Working on it. I guess I needed to build the project to catch that before committing.